src/screens/onboarding/components/step-content/user-content.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Formik } from 'formik';
+import { ScrollView, View } from 'react-native';
+import { Avatar } from 'react-native-paper';
+
+import { AppColor, HeadingLevel } from 'src/common/enums';
+import { FormInput, Heading } from 'src/components';
+import { userInfoValidationSchema } from 'src/validation-schemas';
+import styles from './styles';
+
+const initialValues = {
+  firstName: '',
+  lastName: '',
+  position: '',
+};
+
+const UserContent: React.FC = () => {
+  return (
+    <View style={styles.content}>
+      <ScrollView showsVerticalScrollIndicator={false}>
+        <Heading style={styles.header} level={HeadingLevel.H4}>
+          Tell Us About Yourself
+        </Heading>
+        <View style={styles.imageContainer}>
+          <Avatar.Icon
+            style={styles.userAvatar}
+            size={152}
+            color={AppColor.WHITE}
+            icon="account"
+          />
+        </View>
+        <Formik
+          initialValues={initialValues}
+          validationSchema={userInfoValidationSchema}
+          onSubmit={() => {
+            // TODO
+          }}
+        >
+          <>
+            <FormInput
+              style={styles.input}
```

Comment:
Use this style for containerStyle to avoid this 
![Screenshot 2022-02-17 at 11 44 48](https://user-images.githubusercontent.com/64521658/154449116-6ac179f8-65cd-4278-b0c1-0183a6f439f3.png)

```suggestion
              containerStyle={styles.input}
```